domain/src/main/kotlin/no/nav/su/se/bakover/domain/Behandling.kt

```diff
@@ -83,6 +83,7 @@ data class Behandling(
         BehandlingsStatus.VILKÅRSVURDERT_INNVILGET -> Vilkårsvurdert().Innvilget()
         BehandlingsStatus.VILKÅRSVURDERT_AVSLAG -> Vilkårsvurdert().Avslag()
         BehandlingsStatus.BEREGNET -> Beregnet()
```

Comment:
Kan kanskje rename denne til BEREGNET_INNVILGET for å være konsekvent?

Reply:
done!